migrator/migrations/m_200_to_m_201_compliance_v2_for_4_5/schema/new/compliance_operator_profile_v2.go

```diff
@@ -0,0 +1,58 @@
+package new
```

Comment:
How do you keep track of the columns necessary to migrate?

Reply:
I did a diff of 4.4 `compliance_operator_v2.proto` with the current version.